front/src/actions/calendar.js

```diff
@@ -9,9 +9,12 @@ function createActions(store) {
       try {
         const events = await state.httpClient.get('/api/v1/calendar/event', {
           from,
-          to
+          to,
+          shared: true
+        });
+        const calendars = await state.httpClient.get('/api/v1/calendar', {
+          shared: true
```

Comment:
Are you sure we need to explicitly request for shared calendars?
Shouldn't it the default behavior? "get mines AND shared"?

Reply:
In my opinion, default should be yours only. For example in configuration we fetch only your calendars to select those to sync and It.
Discussion is open, I can change it

Reply:
I agree with @bertrandda, what I see: 

- By default on the configuration page, you should only be able to change your calendars (not the shared one)
- On the calendar page, all calendars (yours + shared) are displayed
- In scenes, only shared calendars are displayed (not in this PR)